edocumentation-transfer-obj/src/main/java/org/dcsa/edocumentation/transferobjects/BookingActiveReeferSettingsTO.java

```diff
@@ -86,6 +86,18 @@ record BkgSuperFreezerTO(
     @NotNull
     boolean isHighValueCargo,
 
+    @NotNull
+    boolean isCargoProbe1Required,
```

Comment:
I don't think @ NotNull on a primitive type does anything, since primitives are by design not null. I always use Boolean for fields I want to explicitly check.

Reply:
changed to boxed types